src/PaymentGateways/Gateways/Stripe/Helpers/CheckoutHelper.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Give\PaymentGateways\Gateways\Stripe\Helpers;
+
+/**
+ * @unreleased
+ * @TODO We should eventually migrate this client-side redirect to a server-side redirect using the url property of the session.
+ */
+class CheckoutHelper
```

Comment:
@JasonTheAdams pointed out avoiding the word "helper" recently. Let's change this to something more descriptive.

Reply:
Resolved, see https://github.com/impress-org/givewp/pull/6199/commits/1af72b1f11fe7de0fca7f791a25e179509f11c23